packages/cli/src/util/oauth.ts

```diff
@@ -393,6 +412,7 @@ export async function verifyJWT(
       issuer: 'https://vercel.com',
       audience: ['https://api.vercel.com', 'https://vercel.com/api'],
     });
+    if (!payload.exp) throw new Error('Missing `exp` claim in JWT');
```

Comment:
This encodes a requirement on our token format that need not necessarily hold. Can we remove it?

Also, I think it would make since to relax this function as much as possible. I.e., just `jwtDecode` not `jwtVerify`. WDYT?